packages/web-components/fast-foundation/src/horizontal-scroll/README.md

```diff
@@ -0,0 +1,125 @@
+---
+id: horizontal-scroll
+title: fast-horizontal-scroll
+sidebar_label: horizontal-scroll
+custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-foundation/src/horizontal-scroll/README.md
+---
+
+An implementation of a content scroller as a web-component.
```

Comment:
I'm wondering if we should add docs here with regard to what to expect in terms of behavior. For instance, the reason that flippers are typically hidden by default is that they are duplicate content when items within the scroll region contain interactive content. If there's only visual or presentational content within this, flippers should be exposed.

Reply:
See comment above.